.typespec/chat/models.tsp

```diff
@@ -681,9 +683,11 @@ model CreateChatCompletionStreamResponse {
     /** Log probability information for the choice. */
     logprobs?: {
       /** A list of message content tokens with log probability information. */
+      @visibility("read")
       content: ChatCompletionTokenLogprob[] | null;
 
       /** A list of message refusal tokens with log probability information. */
+      @visibility("read")
```

Comment:
I could not add the decorator in a client.tsp file for properties on anonymous models due to https://github.com/microsoft/typespec/issues/4818